client/overview/task-list/po-tasks.js

```diff
@@ -0,0 +1,126 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies.
+ */
+import strings from './strings';
+
+const PROGRESSIVE_ONBOARDING_KYC_TPV_LIMIT = 5000;
+
+export const getVerifyBankAccountTask = ( {
+	poEnabled,
+	poComplete,
+	tpv,
+	firstPaymentDate,
+	createdDate,
+} ) => {
```

Comment:
I'm curious about it, any reason to not use TS for this file?

Apart from that, I wonder if we should get those values directly from `wcpaySettings`, instead of passing them from `getTasks`. Because the latter deconstruct them just to pass along to the former 🤔 It will also simplify tests boilerplate.

Reply:
I just followed `getTasks` implementation, did not think about ts 🙈 
Fixed in https://github.com/Automattic/woocommerce-payments/pull/5569/commits/a380f222774d3bf0f3406bc6d08e112abe768923. 
Now need to fix tests!

Reply:
Fixed tests in https://github.com/Automattic/woocommerce-payments/pull/5569/commits/2ae80293b3120fa411d9cbad2dcedc25a95454f2